tests/kernel/boot_ltp.pm

```diff
@@ -153,6 +131,93 @@ EOF
     script_run 'grep -e Huge -e PageTables /proc/meminfo';
     script_run 'echo 1 > /proc/sys/vm/nr_hugepages';
     script_run 'grep -e Huge -e PageTables /proc/meminfo';
+
+    # If the command file (runtest file) is set then we dynamically schedule
+    # the test and shutdown modules.
+    return unless $cmd_file;
+
+    my $test_result_export = {
```

Comment:
It might be better to split off test loading into a separate function. But otherwise LGTM. Feel free to merge this as is.